pkg/storage/streams/store.go

```diff
@@ -63,7 +63,7 @@ type streamStore struct {
 	segmentSize  int64
 	rootKey      *storj.Key
 	encBlockSize int
-	cipher       storj.Cipher
+	Cipher       storj.Cipher
```

Comment:
if we need the Cipher exported, could we make a method a `streamStore.GetCipher()`?